src/indicators/renderers/macd.js

```diff
@@ -0,0 +1,67 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.indicators.renderers.macd = function() {
+
+        var xScale = d3.time.scale(),
+            yScale = d3.scale.linear(),
+            xValue = function(d) { return d.date; };
+
+        var macdLine = fc.series.line(),
+            signalLine = fc.series.line(),
+            divergenceBar = fc.series.bar(),
+            multiSeries = fc.series.multi();
+
+        var macd = function(selection) {
+
+            macdLine
+                .xValue(xValue)
+                .yValue(function(d, i) { return d.macd.macd; })
```

Comment:
Is it worth adding an intermediary accessor for the root data value? i.e. d.macd.signal would be (ignore naming) root(d).signal